beacon-chain/flags/base.go

```diff
@@ -48,4 +48,11 @@ var (
 		Name:  "grpc-gateway-port",
 		Usage: "Enable gRPC gateway for JSON requests",
 	}
+
+	// MinimumHandshake specifies the required number of handshakes in order
+	// to start syncing with external peers.
+	MinimumHandshake = cli.IntFlag{
+		Name:  "minimum-handshakes",
```

Comment:
I agree with Jim’s feedback about handshake being an internal name. Can you choose a better name?